### ISSUE TITLE
Setting Keyboard Shortcut Scroll to Bottom/Top on Mac to Cmd+End/Cmd+Home has no effect

### ISSUE THREAD

**Issue by robdev2** [Microsoft/vscode#46952]
Going to Keyboard shortcuts and changing the key combination for Scroll to Bottom/Top registers the correct key combination, but in the editor it does nothing.

For Scroll to Bottom I see the following key codes when I highlight over the combination

code: End, keyCode: 35, key: End => UI: Command+End, user settings: cmd+end; dispatch: meta+[End]

- VSCode Version: 1.21.1
- OS Version: macOS High Sierra V 10.13.3

Steps to Reproduce:

1. Go to Code -> Preferences -> Keyboard Shortcuts
2. Go to 'Scroll to Bottom'
3.  Double-click the shortcut and change to Cmd+End
4.  Go back to the editor and notice that pressing Cmd+End doesn't do anything

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
That action is in the terminal. Are you looking at the editor instead of the terminal?

Also, it seems like cmd+end is the default on mac?

![image](https://user-images.githubusercontent.com/323878/38167537-7183a9e8-34eb-11e8-962a-b8a49b29f785.png)

**Comment by robdev2** [comment]
What you are showing is definitely the one I was looking at.

It is indeed set to Cmd+End by default... I was just trying to show that it registers the right keystroke combination for the shortcut when setting it manually.

To be honest it doesn't seem to matter what I set it to.  I've tried Alt-End, Ctrl-End, etc... they all don't work.  The only thing that seems to scroll to the bottom of the page is cmd-(down arrow).
